### PR TITLE
[12.0][FIX] update delivery with multiple lines fails

### PR DESCRIPTION
**Pull request by sergiocorato**
Add only a check to avoid possible errors not managed.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
But this is not possible in standard AFAIK...

**Comment by sergiocorato** [comment]
> But this is not possible in standard AFAIK...

A delivery line is created for every OUT related to SO (backorder), is it not normal?

**Comment by pedrobaeza** [comment]
Right, but then the thing should be to catch in the context all the discounts.

File: delivery_auto_refresh/models/sale_order.py [hunk 1]
```diff
@@ -33,8 +34,12 @@ def write(self, vals):
         res = super(SaleOrder, self).write(vals)
         for order in self:
             delivery_line = order.order_line.filtered('is_delivery')
+            digits = self.env.user.company_id.currency_id.decimal_places
```
Comment (rousseldenis) [inline delivery_auto_refresh/models/sale_order.py]: No, you have to use decimal defined on the discount field: dp.get_precision('Discount')

Comment (sergiocorato) [reply delivery_auto_refresh/models/sale_order.py]: Done, thanks!

**Comment by sergiocorato** [comment]
@pedrobaeza @rousseldenis I've modified this PR adding only a clear check, as it's not a common bug probably, so this can be mergeable.

**Comment by pedrobaeza** [comment]
Please check Travis

**Comment by sergiocorato** [comment]
@pedrobaeza done, thanks

**Review by pedrobaeza — APPROVED** [review]

**Review by marcelsavegnago — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]

**Pull request merged**